client/src/components/SortDirection/SortDirection.tsx

```diff
@@ -27,6 +30,7 @@ export const SortDirection = ({
       sortDirection === "ASC" ? "a11y_sort_asc" : "a11y_sort_desc"
     )}
     aria-pressed={active}
+    onClick={() => onDirectionClick(sortDirection === "DESC")}
```

Comment:
How about something like `onDirectionClick(active ? sortDirection === "DESC" | sortDirection !== "DESC" )`, so that clicking on the active sort option can still toggle, rather than do nothing? Minor, but I think it's a little better.